modules/sno-du-configuring-the-operators.adoc

```diff
@@ -0,0 +1,120 @@
+// Module included in the following assemblies:
+//
+// *scalability_and_performance/sno-du-deploying-clusters-on-single-nodes.adoc
+
+:_content-type: PROCEDURE
+[id="sno-du-configuring-the-operators_{context}"]
+= Configuring the Operators
+
+This configuration is provided to add the Operators needed to configure the platform post
+installation. It adds the Local storage Operator, Logging Operator, Performance addon Operator,
+PTP Operator and the SRIOV Network Operator.
+No customization of these files is required. They are configured as follows:
+
+.Local storage Operator
+
+[source,yaml]
+----
+apiVersion: v1
+kind: Namespace
+metadata:
+    annotations:
+        workload.openshift.io/allowed: management
+    name: openshift-local-storage
+---
+apiVersion: operators.coreos.com/v1
+kind: OperatorGroup
+metadata:
+    name: openshift-local-storage
+    namespace: openshift-local-storage
+spec:
+    targetNamespaces:
+        - openshift-local-storage
+----
+
+.Logging Operator
+
+[source,yaml]
+----
+apiVersion: v1
+kind: Namespace
+metadata:
+    annotations:
+        workload.openshift.io/allowed: management
+    name: openshift-logging
+---
+apiVersion: operators.coreos.com/v1
+kind: OperatorGroup
+metadata:
+    name: cluster-logging
+    namespace: openshift-logging
+spec:
+    targetNamespaces:
+        - openshift-logging
+----
+
+.Performance addon Operator
+
+[source,yaml]
+----
+apiVersion: v1
+kind: Namespace
+metadata:
+    annotations:
+        workload.openshift.io/allowed: management
+    labels:
+        openshift.io/cluster-monitoring: "true"
+    name: openshift-performance-addon-operator
+spec: {}
+---
+apiVersion: operators.coreos.com/v1
+kind: OperatorGroup
+metadata:
+    name: performance-addon-operator
+    namespace: openshift-performance-addon-operator
+----
+
+.PTP Operator
+
+[source,yaml]
+----
+apiVersion: v1
+kind: Namespace
+metadata:
+    annotations:
+        workload.openshift.io/allowed: management
+    labels:
+        openshift.io/cluster-monitoring: "true"
+    name: openshift-ptp
+---
+apiVersion: operators.coreos.com/v1
+kind: OperatorGroup
+metadata:
+    name: ptp-operators
+    namespace: openshift-ptp
+spec:
+    targetNamespaces:
+        - openshift-ptp
+----
+
+.SRIOV Network Operator
+[source,yaml]
+----
+apiVersion: v1
+kind: Namespace
+metadata:
+    annotations:
+        workload.openshift.io/allowed: management
+    labels:
+        openshift.io/run-level: "1"
```

Comment:
I don't think this is required

Reply:
Are you saying delete line 109 or the entire labels section?

Reply:
yes

Reply:
Which?

Reply:
removed the labels section